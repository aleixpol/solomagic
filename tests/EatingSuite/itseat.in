\ref something0
\mt somethingsomething ana somethingsomething

\ref something1
\mt somethingsomething banana somethingsomething
