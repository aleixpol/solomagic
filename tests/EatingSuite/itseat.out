\ref something0
\mt somethingsomething ana somethingsomething
\ma somethingsomething angna somethingsomething

\ref something1
\mt somethingsomething banana somethingsomething
\ma somethingsomething banana somethingsomething
